src/main.c

```diff
@@ -150,13 +151,39 @@ static void create_calendars(FocalApp* fm)
 	}
 }
 
+static void update_header_subtitle(FocalApp* fm)
+{
+	WeekView* wv = FOCAL_WEEK_VIEW(fm->weekView);
+
+	struct tm day_start = week_view_get_start(wv);
+	char start[24];
+	strftime(start, sizeof(start), "%e. %B %G", &day_start);
+
+	struct tm day_end = week_view_get_end(wv);
+	char end[24];
+	day_end.tm_hour -= 2;
```

Comment:
The intention of this line isn't clear. If it's really necessary, a comment explaining it should be added